Sources/TuistGenerator/GraphViz/GraphToGraphVizMapper.swift

```diff
@@ -96,7 +99,11 @@ extension GraphDependency {
     fileprivate func isExternal(root: AbsolutePath) -> Bool {
```

Comment:
When is this needed? Shouldn't we use the `isExternal` flag instead of the path?